packages/eslint-plugin/src/rules/no-misused-spread.ts

```diff
@@ -0,0 +1,46 @@
+import { type TSESLint, type TSESTree } from '@typescript-eslint/utils';
+
+import { createRule, getParserServices } from '../util';
+
+type MessageIds = 'forbiddenFunctionSpread';
+
+export default createRule<[], MessageIds>({
+  defaultOptions: [],
+  name: 'no-misused-spread',
+  meta: {
+    docs: {
+      description:
+        "Disallow spread operator that shouldn't be spread most of the time",
```

Comment:
[Proofreading] "most of the time" is kind of an imprecise, vague statement. Users may not understand it.

Instead, let's speak to what this rule is specifically trying to avoid: spreads that don't do anything.

```suggestion
        "Disallow using the spread operator on values that can't be spread",
```